example_audit.rules

```diff
@@ -88,20 +88,46 @@ w /etc/fluent/ -p wa -k syslog
 -w /etc/cron.weekly/ -p wa -k cron
 -w /etc/crontab -p wa -k cron
 -w /var/spool/cron/root -p wa -k cron
--w /var/spool/atspool
--w /etc/at.allow
--w /etc/at.deny
+-w /var/spool/cron -p wa -k cron
+-w /var/spool/atspool -p wa -k cron
+-w /etc/at.allow -p wa -k cron
+-w /etc/at.deny -p wa -k cron
+-w /etc/anacrontab -p wa -k cron
  
 # Modify startup
+-w /boot -p wa -k init
 -w /etc/rc.d/init.d/ -p wa -k init
 -w /sbin/init -p wa -k init
 -w /etc/inittab -p wa -k init
 -w /etc/systemd/ -p wa -k init
+-w /etc/systemd/system -p wa -k init
+-w /etc/systemd/user -p wa -k init
+-w /etc/systemd/network -p wa -k init
+-w /etc/systemd/system/multi-user.target.wants -p wa -k init
+-w /usr/lib/systemd/system -p wa -k init
 -w /etc/init.d/ -p wa -k init
+-w /etc/rc0.d -p wa -k init
+-w /etc/rc1.d -p wa -k init
+-w /etc/rc2.d -p wa -k init
+-w /etc/rc3.d -p wa -k init
+-w /etc/rc4.d -p wa -k init
+-w /etc/rc5.d -p wa -k init
+-w /etc/rc6.d -p wa -k init
+-w /etc/rc.d -p wa -k init
```

Comment:
/etc/rc.d should be recursive already
also given the amount of entries, should this just be watching `/etc` ? ;-)
**each** entry slows down syscall entry latency for writes and attribute change to any file

Reply:
Removed